queries.sql

```diff
@@ -8,3 +8,26 @@
  SELECT *  FROM animals  WHERE neutered = true;
  SELECT *  FROM animals  WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 and weight_kg <= 17.3;
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species = '';
+COMMIT;
+BEGIN TRANSACTION;
+TRUNCATE animals;
+ROLLBACK;
+BEGIN TRANSACTION;
+DELETE FROM animals WHERE date_of_birth > '01-01-2022';
+SAVEPOINT delete_date;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO SAVEPOINT delete_date;
+UPDATE animals SET weight_kg =weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+SELECT name, MIN(weight_kg) min_weight, MAX(weight_kg) max_weight FROM animals GROUP BY name;
+SELECT name, AVG(escape_attempts) avg_escape_attempts FROM animals WHERE date_of_birth BETWEEN '01-01-1990' AND '12-31-2000' GROUP BY name;
```

Comment:
- [ ] Please note that we are looking for the average number of escape attempts per **animal type**, So it seems like the right way to implement this query would be by grouping the animal per type. Therefore I suggest you group the animals by the `species_id` column instead of the name

Reply:
@LionRouge1 Thank you for your feedback Please clarify what you mean here as I do not understand the comment. There is no species_id column in the table